cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -6,85 +6,100 @@ set -exuo pipefail
 make -C ../gocli container
 
 CI=${CI:-"false"}
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 RUN_KUBEVIRT_CONFORMANCE=${RUN_KUBEVIRT_CONFORMANCE:-"true"}
 
 provision_dir="$1"
 provider="${provision_dir}"
 
-function cleanup {
-  cd "$DIR" && cd ../..
-  make cluster-down
+function cleanup() {
+    cd "$DIR" && cd ../..
+    make cluster-down
+}
+
+function delete_ip_reconciler_pods() {
```

Comment:
It may make sense to either fetch all unready pods and filter out pods which we don't care about, or to just patch the yaml in our repo by script when updating manifests.

A delete would be very specific to whereabouts and I have the feeling that other solutions would work in a more general fashion by similar complexity.